cns/restserver/util.go

```diff
@@ -382,32 +529,41 @@ func (service *HTTPRestService) getNetworkContainerResponse(
 
 		logger.Printf("pod info %+v", podInfo)
 
-		containerID, exists = service.state.ContainerIDByOrchestratorContext[podInfo.Name()+podInfo.Namespace()]
+		ncidBytes, err := json.Marshal(service.state.ContainerIDByOrchestratorContext[podInfo.Name()+podInfo.Namespace()])
```

Comment:
why do you need to marshal? can you not check if the Set is non empty? You can expose a func to the Set that you added to return if isEmpty()